src/main/java/ana_raetcaia/task_seven/ArithmeticTest.java

```diff
@@ -0,0 +1,13 @@
+package ana_raetcaia.task_seven;
+
+public class ArithmeticTest {
+    public static void main(String[] args) {
+        Arithmetic divineObj = new Arithmetic();
+        try {
+            System.out.println(divineObj.divideNumbers(65, 0));
```

Comment:
why do we need to create object of Arithmetic to use divide method ?
can we do it without creating object ?

Reply:
If method divineNumbers will be static I can call him directly from class Arithmetic without creating the object

Reply:
so that is the goal
make it static
as an example class math max method is static too